Makefile

```diff
@@ -183,7 +180,7 @@ cluster/cleanup/crds:
 deploy/integreatly-installation-cr.yml: export SELF_SIGNED_CERTS := true
 deploy/integreatly-installation-cr.yml: export INSTALLATION_NAME := integreatly-operator
 deploy/integreatly-installation-cr.yml: export INSTALLATION_TYPE := managed
-deploy/integreatly-installation-cr.yml: export INSTALLATION_PREFIX := rhmi
+deploy/integreatly-installation-cr.yml: export INSTALLATION_PREFIX := integreatly-
```

Comment:
Should we stick to one or the other here? https://github.com/integr8ly/integreatly-operator/pull/323/files#diff-a35ad113285fb0512c183bad2b0aeaebR7